com.github.dariasteam.cowsrevenge.json

```diff
@@ -0,0 +1,113 @@
+{
+  "app-id": "com.github.dariasteam.cowsrevenge",
+  "branch": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cowsrevenge",
+  "finish-args": [
+     "--socket=x11",
+     "--socket=pulseaudio",
+     "--persist=.godot",
+     "--share=ipc",
+     "--device=dri"
+  ],
+  "build-options" : {
+    "split" : false,
+    "no-debuginfo" : true
+  },
+  "modules": [
+    {
+      "name": "scons",
+      "buildsystem": "simple",
+      "cleanup": [ "*" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/scons/scons/2.5.1/scons-2.5.1.tar.gz",
+          "sha256": "0b25218ae7b46a967db42f2a53721645b3d42874a65f9552ad16ce26d30f51f2"
+        }
+      ],
+      "build-commands": [ "python setup.py install --prefix=/app" ]
+    },
+    "shared-modules/glu/glu-9.0.0.json",
+    {
+      "name": "godot",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/godotengine/godot",
+          "commit": "a524cc92e7cccd012d028d870e4648c59473d04e",
+          "branch": "2.1"
+        }
+      ],
+      "build-commands": [
+          "scons platform=server \\
```

Comment:
Why did you remove the options the other one used? Using host dependencies *should* just be a positive.

Reply:
I was trying to find a bug and decided to use the minimal setup I'm familiar with. I totally forgot to rewrite all flags.